--- build/libtorrent/libtorrent-rasterbar-0.16.16/src/file.cpp	2014-02-23 20:14:58.000000000 +0100
+++ file.cpp	2014-05-08 21:19:41.588066535 +0200
@@ -86,8 +86,14 @@
 
 #ifdef TORRENT_LINUX
 // linux specifics
-
-#include <sys/statvfs.h>
+#ifndef ANDROID
+#  include <sys/statvfs.h>
+#else
+#  include <sys/syscall.h>
+#  include <sys/vfs.h>
+#  define statvfs statfs
+#  define fstatvfs fstatfs
+#endif
 #include <sys/ioctl.h>
 #include <linux/types.h>
 #ifdef HAVE_LINUX_FIEMAP_H
@@ -126,7 +132,9 @@
 // related functions support 64-bit offsets.
 // this test makes sure lseek() returns a type
 // at least 64 bits wide
+#ifndef ANDROID
 BOOST_STATIC_ASSERT(sizeof(lseek(0, 0, 0)) >= 8);
+#endif
 
 #endif // posix part
 
