--- build/libtorrent/libtorrent-rasterbar-0.16.16/src/utp_stream.cpp	2014-02-23 20:14:58.000000000 +0100
+++ utp_stream.cpp	2014-05-08 23:36:10.984066535 +0200
@@ -40,6 +40,10 @@
 #include "libtorrent/random.hpp"
 #include <boost/cstdint.hpp>
 
+#ifndef INT64_MAX
+# define INT64_MAX (__INT64_C(9223372036854775807))
+#endif
+
 #define TORRENT_UTP_LOG 0
 #define TORRENT_VERBOSE_UTP_LOG 0
 #define TORRENT_UT_SEQ 1
